documentation/HpdidLookup-API.md

```diff
@@ -176,3 +176,6 @@ Lastly, due to privacy issues, in the very rare cases that a PRIME enrollee has
 |Independent User - without OBOs|
 |On-behalf-of User|
 |On-behalf-of User – Pharmacy|
+|Device Provider Agent - with OBOs|
+|On-behalf-of User – Device Provider|
```

Comment:
There is something strange with this hyphen ... doesn't match the one in `AgreementVersion.cs`

Reply:
It is fixed.